app/services/geographic_builder.rb

```diff
@@ -48,12 +48,13 @@ def build_place(node)
   def place_from_code(node)
     return [] unless node.code && node.source
 
-    code = node.code.sub(/[^\w-]/, '') # remove any punctuation (except dash).
+    code_with_dashes = node.code.gsub(/[^\w-]/, '') # remove any punctuation (except dash).
+    code_without_dashes = code_with_dashes.sub(/-+$/, '') # remove trailing dashes.
     case node.source.code
     when 'marcgac'
-      [Marc::Vocab::GeographicArea.fetch(code)]
+      [Marc::Vocab::GeographicArea::TERMS[code_with_dashes] || Marc::Vocab::GeographicArea::TERMS[code_without_dashes]].compact
```

Comment:
Please use `fetch` with a default (`nil`) here rather than addressing the constants (which is an implementation detail).

Reply:
Duh.

Reply:
Oh wait, it doesn't take a default.

Reply:
`bundle update marc-vocab`

Reply:
The 0.3.0 version does.